packages/sui-ssr/README.md

```diff
@@ -70,6 +70,29 @@ It use the stdout stream so you can do things like:
   $ sui-ssr archive > ./myFileNameOrWhatever.zip
 ```
 
+## Hooks
+
+If you want change the server´s behavior for very specific business operation like handler error or logging. You must to use the hooks concept.
+
+To do that define a file `src/hooks.js` this file must be something like this:
+
+```
+import TYPES from '@s-ui/ssr/hooks-types'
+
+export default {
+  [TYPES.LOGGING]: (req, res, next) => {
+    console.log(req.url)
+    next()
+  }
+}
+```
+
+Here we are define a logger to the console directly. Each hook type could be or a middleware function or an array of middlewares functions.
```

Comment:
we implement instead of we are define, I would even use something like: here we implement a direct log into the console or something like that.